support/crown/src/unrooted_must_root.rs

```diff
@@ -331,7 +336,9 @@ impl<'a, 'tcx> visit::Visitor<'tcx> for FnDefVisitor<'a, 'tcx> {
                     cx.lint(
                         UNROOTED_MUST_ROOT,
                         format!("Expression of type {:?} must be rooted", ty),
-                        |lint| lint.set_span(pat.span),
+                        |lint| {
```

Comment:
This change looks unrelated?

Reply:
`.with_span` (or what was before `.set_span`) started to return `&mut Self` while closure expects `()` for return type.